README.md

```diff
@@ -69,6 +82,18 @@ warn_on_mons_with_no_ac = 2
 #
 # Set to 0 to disable.
 bluetooth_low_pct = 40
+
+# Set to false to disable state change notifications e.g. when charging, discharging, reaching the threshold
+state_notif_enabled = true
```

Comment:
This should be an enum showing which state change notifications should be shown. See below for more about that.

Reply:
Like a single config keyword that is a list/array?

Reply:
Ditto directly above :-)

Reply:
Marking this as resolved because I think it might already be done, see latest comment at the bottom.